### PR TITLE
Feature zip archives and checksum UI and controller

### PR DESCRIPTION
**Pull request by edsonarios**

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/view/Models/VideoRequestForm.java [hunk 1]
```diff
@@ -26,6 +26,8 @@
     private final String height = "height";
     private final String audio = "audio";
     private final String metadata = "metadata";
+    private final String checksum = "checksum";
+
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/view/Models/VideoRequestForm.java]: Delete blank line

File: src/main/java/org/fundacion/jala/converter/controller/VideoConverterController.java [hunk 2]
```diff
@@ -46,13 +60,83 @@ public String uploadFile(final @RequestParam("file") MultipartFile file,
                              final @RequestParam("width") int width,
                              final @RequestParam("height") int height,
                              final @RequestParam("audio") boolean audio,
+                             final @RequestParam("checksum") String checksum,
                              final @RequestParam("metadata") String metadata
-    ) throws IllegalStateException, IOException {
-        String outputFilename = ConverterFacade.getVideoConverter(new VideoParameter(fileStorageService.uploadFile(file), outputFormat,
+    ) throws IllegalStateException, IOException, InterruptedException {
+        final int waitTime = 6000;
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/controller/VideoConverterController.java:66]: You shouldn't use magic numbers

File: src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java [hunk 3]
```diff
@@ -1,6 +1,6 @@
 /**
  * Copyright (c) 2021 Fundacion Jala.
- *
+ * <p>
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java]: Delete this extra char

File: src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java [hunk 4]
```diff
@@ -73,58 +78,70 @@ public VideoConverterInterface() {
...
+            videoRequestForm.addThumbnail(String.valueOf(settings.isThumbnailRequired()));
+            videoRequestForm.addFrameRate(settings.getFrame());
+            videoRequestForm.addWidth(settings.getWidthResolution());
+            videoRequestForm.addHeight(settings.getHeightResolution());
+            videoRequestForm.addAudio(String.valueOf(settings.isAudioSelected()));
+            videoRequestForm.addResolution(settings.getWidthResolution());
+            videoRequestForm.addChecksum(checksumLocal);
+            videoRequestForm.addMetadata(String.valueOf(settings.isMetadataRequired()));
+
+
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java]: Delete blank lines

File: src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java (hunk 3 above)
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java]: This should be deleted

File: src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java [hunk 5]
```diff
@@ -14,6 +14,7 @@
 import org.fundacion.jala.converter.view.Models.VideoRequestForm;
 import org.fundacion.jala.converter.view.controllers.ClientRequest;
 import org.fundacion.jala.converter.view.utilities.JLabelStyle;
+
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java]: This blank line should be deleted

File: src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java [hunk 6]
```diff
@@ -23,7 +24,10 @@
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
 import java.io.IOException;
+import java.security.NoSuchAlgorithmException;
+
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java]: This blank line should be deleted

File: src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java [hunk 7]
```diff
@@ -73,58 +78,70 @@ public VideoConverterInterface() {
...
+            videoRequestForm.addOutputFormat(menuConverterType.getConvertTo());
+            videoRequestForm.addThumbnail(String.valueOf(settings.isThumbnailRequired()));
+            videoRequestForm.addFrameRate(settings.getFrame());
+            videoRequestForm.addWidth(settings.getWidthResolution());
+            videoRequestForm.addHeight(settings.getHeightResolution());
+            videoRequestForm.addAudio(String.valueOf(settings.isAudioSelected()));
+            videoRequestForm.addResolution(settings.getWidthResolution());
+            videoRequestForm.addChecksum(checksumLocal);
+            videoRequestForm.addMetadata(String.valueOf(settings.isMetadataRequired()));
+
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java]: This blank line should be deleted

File: src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java (hunk 4 above)
Comment (cristianfchq) [reply src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java]: I agree with you

File: src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java (hunk 7 above)
Comment (cristianfchq) [reply src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java]: I agree with you Dani

**Review by danielaSantaCruz — APPROVED** [review]

**Review by Paola-Xaq — APPROVED** [review]

**Review by cristianfchq — APPROVED** [review]

**Pull request merged**